ast/ddl.go

```diff
@@ -547,21 +548,99 @@ func (n *TableToTable) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+// ViewAlgorithm is VIEW's SQL AlGORITHM characteristic
+// See https://dev.mysql.com/doc/refman/5.7/en/view-algorithms.html
+type ViewAlgorithm int
+
+const (
+	AlgorithmUndefined ViewAlgorithm = iota
+	AlgorithmMerge
+	AlgorithmTemptable
+)
+
+func (v *ViewAlgorithm) String() string {
+	switch *v {
+	case AlgorithmMerge:
+		return "MERGE"
+	case AlgorithmTemptable:
+		return "TEMPTABLE"
+	case AlgorithmUndefined:
+		return "UNDEFINED"
+	}
+	return "UNDEFINED"
+}
+
+// ViewSecurity is VIEW's SQL SECURITY characteristic
+// See https://dev.mysql.com/doc/refman/5.7/en/create-view.html
+type ViewSecurity int
+
+const (
+	SecurityDefiner ViewSecurity = iota
+	SecurityInvoker
+)
+
+func (v *ViewSecurity) String() string {
+	switch *v {
+	case SecurityDefiner:
+		return "DEFINER"
+	case SecurityInvoker:
+		return "INVOKER"
+	}
+	return "DEFINER"
+}
+
+// ViewCheckOption is VIEW's WITH CHECK OPTION clause part
```

Comment:
Add a `.` at the end of this comment.
So as the other comment lines if they are complete sentences.